be/test/io/cache/block_file_cache_test_common.h

```diff
@@ -74,14 +74,21 @@ extern int disk_used_percentage(const std::string& path, std::pair<int, int>* pe
 extern fs::path caches_dir;
 extern std::string cache_base_path;
 extern std::string tmp_file;
-
+#if defined(__clang__)
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wunknown-warning-option"
+#pragma clang diagnostic ignored "-Wdeprecated-literal-operator"
+#endif
 constexpr unsigned long long operator"" _mb(unsigned long long m) {
```

Comment:
dont ignore warning. just fix like:
```suggestion
constexpr unsigned long long operator""_mb(unsigned long long m) {
```